package.json

```diff
@@ -17,7 +18,9 @@
   "scripts": {
     "test": "mocha --harmony",
     "test-cov": "node --harmony node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- --reporter dot",
-    "test-travis": "node --harmony node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- --reporter dot"
+    "test-travis": "node --harmony node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- --reporter dot",
+    "prepublish": "npm run browserify",
+    "browserify": "node_modules/.bin/browserify index.js -o ./co-browser.js -s co"
```

Comment:
`browserify` is available in the `$PATH` if you run something with `npm run` :wink: